doc/api/deprecations.md

```diff
@@ -2518,6 +2518,27 @@ Type: Documentation-only
 Prefer [`response.socket`][] over [`response.connection`] and
 [`request.socket`][] over [`request.connection`].
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: http finished
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/28679
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
+
+[`response.finished`][] indicates whether [`response.end()`] has been
+called and [`response.writableEnded`][] is `true`, not whether the underlying
+data has been flushed, `'finish'` has been emitted and
+[`response.writableFinished`][] is `true`.
```

Comment:
This description is somewhat difficult to read for me. I might not have enough knowledge about the exact functionality but without further context, it seems quite hard to follow.

I understand the first part of the sentence but the second part is not absolute clear to me. Especially due to the `and`. What about:

```
[`response.finished`][] indicates whether [`response.end()`] has been
called and [`response.writableEnded`][] is `true`. It does not indicate
whether the underlying data has been flushed or if `'finish'` has been
emitted.
```

Reply:
`response.writableEnded` should true after `respone.end()`.

Reply:
Ok, but that does not seem immediately clear for me from the current description.

Reply:
Ok, any suggestion on improvement?

Reply:
I am sorry but I do not know the underlying behavior well enough to give a recommendation in this case.

Reply:
How about now?

Reply:
That sounds better to me! Just the `and` still confuses me. Should it not be: `... has been emitted _or_ response.writeableFinished ...`?

Reply:
`writeableFinished` is set to `true` at the same time as `'finish'` is emitted. I can remove one of them to make it more ambiguous and clearer at the same time :D.

Reply:
How about now?